src/System.Linq.Parallel/tests/PlinqDelegateExceptions.cs

```diff
@@ -77,9 +77,9 @@ public static void OrderBy()
         }
 
         /// <summary>
-        /// Heavily exercises OrderBy in the face of user-delegate exceptions. 
+        /// Heavily exercises OrderBy in the face of user-delegate exceptions.
         /// On CTP-M1, this would deadlock for DOP=7,9,11,... on 4-core, but works for DOP=1..6 and 8,10,12, ...
```

Comment:
From the look of it, I haven't done any actual work on this file (just run cleanup over it in batch).  I'm probably going to revert this file for now, in favor of dealing with it more later.